descriptors/ModuleDescriptor-template.json

```diff
@@ -54,7 +54,7 @@
             "perms.users.assign.mutable"
           ],
           "permissionsRequired": [
-            "perms.users.item.delete"
+            "perms.users.item.id.delete"
```

Comment:
Should this be `perms.users.item.perm.delete`? I believe this is the "resource identifier" as specified [here](https://folio-org.atlassian.net/wiki/spaces/FOLIJET/pages/156368925/Permissions+naming+convention) is it not?

Reply:
Ok I see that the `{id}` here is the resource.